packages/dashboard/src/components/pages/platforms/dmt5_synthetic/dmt5-synthetic.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from 'react';
```

Comment:
Is this file not meant to be localized yet?

Reply:
it is meant to be localized on implementation, this is only layout usage example atm, not the full implementation